regen-network/regen-web
# fix: APP-234 create sell order

## Description

https://regennetwork.atlassian.net/browse/APP-234



---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...


### How to test



From https://deploy-preview-2410--regen-marketplace.netlify.app/
Log in with an account that has some ecocredits and go to your portfolio.
Create a new sell order leaving the checkbox "Require that credits are retired upon purchase" unchecked, then go to the storefront page, the purchase options of the sell order should be "tradable" and "retirable".
Create another sell order and check the checkbox, the only purchase option of the sell order should be "retirable".

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...
